### PR TITLE
Set position and width of the `PlaybackPanel`

### PR DESCRIPTION
**Pull request by m-agour**
In this PR, I introduce some improvements to the `PlaybackPanel` by allowing setting its position and width as well as making it responsive to window resizing by default.

https://user-images.githubusercontent.com/63170874/191489960-0ccb0b26-983c-4708-bf27-905afb55307c.mp4

### CODE REVIEW

File: fury/ui/elements.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 import numpy as np
 
 from fury.data import read_viz_icons
-from fury.lib import PolyDataMapper2D
+from fury.lib import PolyDataMapper2D, Command
```
Comment (skoudoro) [inline fury/ui/elements.py:17]: pep8: Can you remove `PolyDataMapper2D` it seems to not be used

File: fury/ui/elements.py [hunk 2]
```diff
@@ -3578,14 +3578,15 @@ class PlaybackPanel(UI):
        such as play, pause, stop, and seek.
     """
 
-    def __init__(self, loop=False, position=(0, 0)):
-        super(PlaybackPanel, self).__init__()
-        self.position = position
+    def __init__(self, loop=False, position=(0, 0), width=None):
```
Comment (skoudoro) [inline fury/ui/elements.py:3581]: I wonder if make sense to have position at `(0,0)` instead of `None`

**Review by skoudoro — COMMENTED** [review]
Hi @m-agour,

Thank you for this feature, it works well. See below for few comment but it should be ready to merge soon

File: fury/ui/elements.py (hunk 2 above)
Comment (m-agour) [reply fury/ui/elements.py:3581]: I think it should be `(0, 0)` to keep consistency with other UI elements that has `position=(0, 0)`. That is true if the position is measured from the bottom left of the panel

Comment (skoudoro) [reply fury/ui/elements.py:3580]: ok, fair enough. LGTM, merging

**Pull request merged**